lib/ansible/module_utils/urls.py

```diff
@@ -550,7 +550,10 @@ def build_ssl_validation_error(hostname, port, paths):
     msg.append('You can use validate_certs=False if you do'
                ' not need to confirm the servers identity but this is'
                ' unsafe and not recommended.'
-               ' Paths checked for this platform: %s')
+               ' Paths checked for this platform: %s.')
+
+    if exc:
+        msg.append('The exception msg was: %s.' % exc)
 
     raise SSLValidationError(' '.join(msg) % (hostname, port, ", ".join(paths)))
```

Comment:
So i'll go with the first suggestion and if you disagree I can spin a new review as i wasn't sure about where to put the to_native myself.

I don't think there's any way for anything else here to be non-string (to_native seems to be a safer, python version independent str()). Really the exc should work without the to_native as well as %s will force the type, but the to_native(e) is used a lot.